tensorrt_llm/_torch/pyexecutor/sampler.py

```
@@ -1992,11 +2004,11 @@ def provision_bias_index() -> int:
 
         # Collect bias information
         req_bias = None
-        for i, (req, steps) in enumerate(zip(requests, request_steps)):
-            steps = int(steps.item())
+        for i, (req, steps) in enumerate(zip(requests, request_steps_list)):
```

Ruff (0.15.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)